test/functional/test_framework/test_framework.py

```diff
@@ -96,6 +96,7 @@ def __init__(self) -> None:
         """Sets test framework defaults. Do not override this method. Instead, override the set_test_params() method"""
         self.chain: str = 'regtest'
         self.setup_clean_chain: bool = False
+        self.noban_tx_relay: bool = False
```

Comment:
I really like this clean up commit! Nice feature. I didn't deeply check all the results from this git grep, but there might still be more to do? Maybe from recent test additions on master?

```
--> git grep whitelist *.py | wc -l
55
```

Reply:
>  there might still be more to do?

Maybe. I'm checking it again, I only modified the tests that explicity points out that is using it to speedup tx relay. so, there are other tests using it for other purposes.